spec/forms/steps/client/contact_details_form_spec.rb

```diff
@@ -49,16 +49,6 @@
   end
 
   describe 'validations' do
-    context 'when telephone_number is blank' do
-      let(:telephone_number) { '' }
```

Comment:
Not a blocker, but we can explicitly check the validity of a blank attribute and to cover against inadvertently reintroducing the presence validation in the future. Example:

```ruby
context 'when telephone_number is blank' do
  it { is_expected.not_to validate_presence_of(:telephone_number) }
end
```